### PR TITLE
Issue#572: send verification mail on account creation

### PR DESCRIPTION
**Pull request by lsenta**

### CODE REVIEW

File: libs/auth/src/lib/+state/auth.service.ts [hunk 1]
```diff
@@ -81,28 +82,33 @@ export class AuthService {
     await this.db.doc<User>(`users/${uid}`).delete();
   }
 
+  public async sendVerifyEmail() {
+    return this.afAuth.auth.currentUser.sendEmailVerification();
+  }
+
   /** Deletes user subCollections */
-  private async _deleteSubCollections (uid) {
+  private async _deleteSubCollections(uid) {
```
Comment (GrandSchtroumpf) [inline libs/auth/src/lib/+state/auth.service.ts]: Please remove "_". private flag is here for that. also type your parameter.

File: libs/auth/src/lib/+state/auth.service.ts [hunk 2]
```diff
@@ -81,28 +82,33 @@ export class AuthService {
...
     // @todo check if user is the only member of org (and the only ADMIN)
     // @todo remove uid from org.userIds
     const permissions = await this._getUserSubcollectionItems(uid, 'permissions');
-    return await permissions.map(o => this.db.doc<User>(`users/${uid}`)
-      .collection('permissions')
-      .doc(o.id)
-      .delete()
+    return Promise.all(
+      permissions.map(o =>
+        this.db
```
Comment (GrandSchtroumpf) [inline libs/auth/src/lib/+state/auth.service.ts]: For consistency: 
```typescript
permissions.map(({id}) => this.db.doc(`users/${uid}/permissions/${id}`).delete())
```

File: libs/auth/src/lib/+state/auth.service.ts (hunk 1 above)
Comment (lsenta) [reply libs/auth/src/lib/+state/auth.service.ts]: C'est bien parcque c'est toi je suis passé dessus pour virer les bugs et fixer le style ;)

File: libs/auth/src/lib/+state/auth.service.ts (hunk 2 above)
Comment (lsenta) [reply libs/auth/src/lib/+state/auth.service.ts]: Idem ;)

File: libs/auth/src/lib/+state/auth.service.ts (hunk 1 above)
Comment (GrandSchtroumpf) [reply libs/auth/src/lib/+state/auth.service.ts]: I know. But we need to clean the code bit by bit. I think it's kind of house keeping in a flatshare ;).
I'm glad you went there to fixe the style so I can see that ! 👍

**Pull request merged**